services/backup_socrata.py

```diff
@@ -0,0 +1,78 @@
+import argparse
+import datetime
+import os
+import requests
+
+import boto3
+
+from config.knack import CONFIG
+import utils
+
+BUCKET = os.getenv("BUCKET")
+AWS_ACCESS_ID = os.getenv("AWS_ACCESS_ID")
+AWS_SECRET_ACCESS_KEY = os.getenv("AWS_SECRET_ACCESS_KEY")
+SOCRATA_APP_TOKEN = os.getenv("SOCRATA_APP_TOKEN")
+
+
+def export_dataset(resource_id):
+    url = f"https://data.austintexas.gov/resource/{resource_id}.csv?$limit=99999999?$$app_token={SOCRATA_APP_TOKEN}"
```

Comment:
Providing app token hopefully reduces the risk of being throttled [docs](https://dev.socrata.com/docs/app-tokens.html). Maybe send as header instead?